pyvo/mivot/features/sky_coord_builder.py

```diff
@@ -47,22 +21,27 @@ class SkyCoordBuilder:
       it contains the information required to compute the epoch propagation which is a major use-case
     '''
 
-    def __init__(self, mivot_instance_dict):
+    def __init__(self, mivot_instance):
         '''
         Constructor
 
         parameters
         -----------
-        mivot_instance_dict: viewer.MivotInstance.to_dict()
-            Internal dictionary of the dynamic Python object generated from the MIVOT block
+        mivot_instance: dict or MivotInstance
+            Python object generated from the MIVOT block as either a Pyhon object or a dict
         '''
-        self._mivot_instance_dict = mivot_instance_dict
+        if isinstance(mivot_instance, dict):
```

Comment:
Again, no point for 2 alternative types for model representation. End users are interested in actual MANGO or other astronomy related objects (SkyCoords) not the model representation. Just my opinion.

Reply:
this is mostly driven by the will of simplifying  unit tests

Reply:
This feature has been removed